compiler/src/iree/compiler/Dialect/HAL/Target/TargetRegistry.cpp

```diff
@@ -156,7 +164,8 @@ bool llvm::cl::parser<TargetRegistryRef>::parse(Option &O, StringRef ArgName,
   // of target backends and create a new registry with just that subset but
   // ownership gets tricky.
   if (Arg != "global")
-    return true;
+    llvm::errs() << "Cannot parse target registery: " << Arg
+                 << ". Defaulting to global.\n";
```

Comment:
style: wrap multi-line expressions in braces (at the point the condition or the body go multi-line the braces are important for both readability, hackability, and correctness)
https://google.github.io/styleguide/cppguide.html#Formatting_Looping_Branching

```suggestion
  if (Arg != "global") {
    llvm::errs() << "Cannot parse target registery: " << Arg
                 << ". Defaulting to global.\n";
  }
```